defaults.go

```diff
@@ -15,13 +15,14 @@ import (
 	"github.com/bullettrain-sh/bullettrain-go-php"
 	"github.com/bullettrain-sh/bullettrain-go-python"
 	"github.com/bullettrain-sh/bullettrain-go-ruby"
+	"github.com/devkanro/bullettrain-go-k8s"
```

Comment:
For this PR, since it will end up living on this repo, please change the path from your own name.

Reply:
When you accept the repo, I will change it